src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheck.java

```diff
@@ -60,6 +60,13 @@
  *   &lt;property name=&quot;format&quot; value=&quot;a href&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void myTest() {
+ *     String test = "a href"; // violation
+ *
+ * }
+ * </pre>
  * <p>
  * To configure the check to forbid leading zeros in an integer literal,
  * other than zero and a hex literal:
```

Comment:
there is no mention that `ignoreCase` property set to true.

May be it makes sense to make a separate example to show `ignoreCase` property usage

Reply:
I am done.